pkg/iris/scan.go

```diff
@@ -53,99 +54,86 @@ const (
 
 const StatusErrorFormat = "Scan failed while posting status. RunId: [%s], with reason: '%v'"
 
-func (s *scanner) Scan(getKubernetesAPI kubernetes.GetKubernetesAPI, config *rest.Config, workspaceId string, configurationName string, accessToken string) error {
-	configuration, err := s.irisApi.GetConfiguration(configurationName, accessToken)
+func (s *scanner) Scan(getKubernetesAPI kubernetes.GetKubernetesAPI, config *rest.Config, configurationName string) error {
+	configuration, err := s.configService.GetConfiguration(configurationName)
```

Comment:
how do we ensure we are getting the config from the right workspace with the configname?

Reply:
Intercepting the acces token in Iris and fetching the workspace id there.
Then workspaceId and configName need to be in the query to fetch the config